ui/utils.py

```diff
@@ -251,4 +254,6 @@ def fit_image_to_screen(image, o):
             top = delta // 2
             bottom = delta - top
         image = ImageOps.expand(image, border=(left, top, right, bottom), fill="black")
+        if (o.width, o.height) != image.size:
+	    logger.debug("Something strange has happened! Please contact LouisPi on the Zerophone IRC for help")
```

Comment:
=D Better throw an exception? People won't read the logs (and with `logger.debug` being the lowest logging level typically not enabled by default, this will not appear in logs anyway). Also, no need to tell people to contact anyone - with each extra work they need to do, there's less and less possibility they will do it. We do have bugreporting and will be adding custom exception hooks to make reporting automatic, don't worry =) In general, very mindful of you to add this - one small thing, for runtime checks like this, it's best if you use `assert`, it will throw an exception by itself too!